site/content/docs/manifest/pipeline.en.md

```diff
@@ -23,6 +23,9 @@ List of all available properties for a Copilot pipeline manifest. To learn more
         stages: 
           - # By default all workloads are deployed concurrently within a stage.
             name: test
+            pre_deployments:
+              db_migration:
+                buildspec: ./buildspec.yml
```

Comment:
```suggestion
                buildspec: ./db-migrate-buildspec.yml
```

Reply:
I kinda wanted to make it the most basic, recognizable format...?